reporter/src/main/kotlin/reporters/NoticeByPackageReporter.kt

```diff
@@ -178,7 +179,7 @@ class NoticeByPackageProcessor(input: ReporterInput) : AbstractNoticeReporter.No
     }
 
     private fun MutableList<() -> String>.addProcessedFindings(processedFindings: LicenseFindingsMap) {
-        processedFindings.entries.forEachIndexed { index, (license, copyrights) ->
+        processedFindings.entries.sortedBy { it.key }.forEachIndexed { index, (license, copyrights) ->
```

Comment:
Is explicit sorting really necessary here? `processedFindings` is of type `LicenseFindingsMap` which is an alias for a `SortedMap`. Getting a bit off-topic: If we wouldn't need the `index` we could also just use `processedFindings.forEach {}` (there seems to be no `forEachIndexed` for `SortedMap`). So Maybe we should rewrite this to use a manually declared index variable outside of the loop.

Reply:
I checked, it's not required to sort here.